api_db.php

```diff
@@ -76,17 +76,18 @@
         $dbquery .= ' status, reply_type, reply_time, dnssec';
         $dbquery .= ' FROM query_storage q';
         $dbquery .= ' WHERE timestamp >= :from AND timestamp <= :until ';
-        if (isset($_GET['types'])) {
-            $types = $_GET['types'];
-            if (preg_match('/^[0-9]+(?:,[0-9]+)*$/', $types) === 1) {
+        if (isset($_GET['status'])) {
+            // if some query status should be excluded
+            $excludedStatus = $_GET['status'];
+            if (preg_match('/^[0-9]+(?:,[0-9]+)*$/', $excludedStatus) === 1) {
                 // Append selector to DB query. The used regex ensures
                 // that only numbers, separated by commas are accepted
                 // to avoid code injection and other malicious things
                 // We accept only valid lists like "1,2,3"
                 // We reject ",2,3", "1,2," and similar arguments
-                $dbquery .= 'AND status IN ('.$types.') ';
+                $dbquery .= 'AND status NOT IN ('.$status.') ';
```

Comment:
You forgot to change it here.

Reply:
I just committed your suggestion online without thinking about it further ....

Reply:
Yeah... github didn't let me select all lines at once, but I tried to warn you on my first comment:

> if the variable was called $excludedStatus (lines 81, 82 **and 88**).

Reply:
I obviously lack comprehensive reading skills...